pmd-core/src/main/java/net/sourceforge/pmd/renderers/internal/sarif/SarifLog.java

```diff
@@ -2484,6 +2527,239 @@ public java.lang.String toString() {
     }
 
 
+    /**
+     * Configure the Sarif reporting defined by a reportingDescriptor.
+     * Can be used as the defaultConfiguration of a reportingDescriptor.
+     * Can also be used in configurationOverride to override those defaults.
+     */
+    public static class ReportingConfiguration {
+        /**
+         * Boolean, to dis- and enable the config based on matching a rule (through the Descriptor).
+         * Default: true.
+         */
+        private boolean enabled;
```

Comment:
For consistency reasons, we should use `Boolean` here (and for the getter, and setter in the builder)